packages/display/src/Stage.js

```diff
@@ -0,0 +1,63 @@
+import Container from './Container';
+import InnerStage from './InnerStage';
+import Runner from 'mini-runner';
+
+/**
+ * Stage is a Container that takes care of add/remove events and animations
```

Comment:
As for me, I don't quite get a reason to have multiple stages.

Would be cool to add short example/use-cases in docs?

Reply:
I think something like your description is a great thing to put here.

Just maybe make it more compact.